src/librustdoc/passes.rs

```diff
@@ -106,7 +106,7 @@ pub fn strip_private(mut krate: clean::Crate) -> plugins::PluginResult {
             retained: &mut retained,
             access_levels: &access_levels,
         };
-        krate = stripper.fold_crate(krate);
+        krate = ImportStripper.fold_crate(stripper.fold_crate(krate));
```

Comment:
Now that this change hit Nightly I’m trying it out but I’m surprised to not see `strip-priv-imports` listed in the default passes in `rustdoc --passes list`. As far as I understand it’s indeed not a default pass, but this line makes it implied by `strip-private`. This is not exactly wrong, but at least confusing.

Maybe add “, implies `strip-priv-imports`” should be added to the description for `strip-private` in `src/librustdoc/lib.rs`?